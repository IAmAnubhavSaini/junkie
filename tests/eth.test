#!/bin/sh

NAME="Ethernet"
PCAPS="pcap/eth/stp_cdp.pcap"
filter() {
	grep '^Ethernet' | cut -d' ' -f 2-
}
OUTPUT="eth.out"

. "$srcdir/dump_test"
