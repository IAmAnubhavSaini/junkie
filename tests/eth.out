-- pcap/eth/stp_cdp.pcap --
head_len=14, payload=38, vlan_id=-1, source=dc:7b:94:de:c1:98, dest=01:80:c2:00:00:00, proto=0x0
head_len=14, payload=34, vlan_id=-1, source=dc:7b:94:de:c1:86, dest=01:00:0c:cc:cc:cc, proto=0x0
